backend/tet/tet/views.py

```diff
@@ -18,14 +18,17 @@ def get(self, request):
                 "given_name": user.first_name,
                 "family_name": user.last_name,
                 "email": user.email,
-                "name": f"{user.first_name} {user.last_name}",
+                "name": f"{user.first_name} {user.last_name}".strip(),
                 "is_ad_login": user.is_staff,
                 "organization_name": get_organization_name(request),
             }
 
             if not (userinfo["given_name"] or userinfo["family_name"]):
```

Comment:
Should this be `if not (userinfo["given_name"] or not userinfo["family_name"]):`

Reply:
!(x or y) = !x and !y i.e. both are empty so it's just checking that there's no name information at all present and then falls back to using email.

Reply:
Oh I didnt see the brackets... perhaps !x and !y could be better if someone else is as blind as me 👓